mkdocs.yml

```diff
@@ -75,7 +75,6 @@ nav:
             - IBFT 2.0: HowTo/Configure/Consensus-Protocols/IBFT.md
             - QBFT: HowTo/Configure/Consensus-Protocols/QBFT.md
             - Clique: HowTo/Configure/Consensus-Protocols/Clique.md
-            - Quorum IBFT 1.0 network: HowTo/Configure/Consensus-Protocols/QuorumIBFT.md
```

Comment:
This page removal requires a redirection to prevent 404.
See end of `mkdocs.yml` for redirect plugin config.
Maybe the best would be to redirect to IBFT2 or QBFT.

Reply:
add a redirect for the page removal